test/feeder_ex_test.exs

```diff
@@ -30,6 +30,11 @@ defmodule FeederExTest do
 
   end
 
+  test "parse! returns just the parsed results on success" do
+    parsed_feed = FeederEx.parse!(File.read!(@sample_file))
+    assert parsed_feed.title == "Liftoff News"
+  end
```

Comment:
would be great if you could add a test for the bad case as well. http://elixir-lang.org/docs/stable/ex_unit/ExUnit.Assertions.html#assert_raise/2 could help.